flow/genericactors.actor.h

```diff
@@ -1226,7 +1225,7 @@ struct FlowLock : NonCopyable, public ReferenceCounted<FlowLock> {
 	// number of permits, and release() makes the caller no longer a holder of the lock. release() only runs waiting take()rs
 	// after the caller wait()s
 
-	struct Releaser : NonCopyable {
+	struct [[nodiscard]] Releaser : NonCopyable {
```

Comment:
Are there any functions returning a `Releaser` that this would affect?

Reply:
I don't think so, but as far as I can tell discarding it would probably be unintentional